docs/overview.md

```diff
@@ -13,22 +16,49 @@ As a versatile solution, ScalarDB supports a range of databases, including:
 
 For details on which databases ScalarDB supports, refer to [Supported Databases](scalardb-supported-databases.md).
 
+## Why ScalarDB?
+
+There are several solutions, such as global transaction managers, data federation engines, and HTAP systems, built towards similar goals, but they are limited in the following perspectives:
+
+- Global transaction managers (e.g., Oracle MicroTx, Atomikos) are designed to run transactions across a limited set of heterogeneous databases.
```

Comment:
Maybe you have already considered it, but we might be able to say something about what the limited set is, e.g., only XA-compliant databases. Or we can say like, "They do not support NoSQL databases".